GithubHandlers.py

```diff
@@ -152,29 +164,54 @@ def get_file_contents(self, filepath: str, ref=None):
             traceback.print_exc()
             return "Could not retrieve file contents."
 
-    def get_pr_deltas(self, pr):
+    def get_pr_deltas(self, pr) -> str:
```

Comment:
When a patch is `None`, ensure to provide a meaningful message to the end user, and test these scenarios.